Application/EdFi.Ods.Features/ChangeQueries/SnapshotContext/SnapshotContextActionFilter.cs

```diff
@@ -47,9 +41,9 @@ public Task OnActionExecutionAsync(ActionExecutingContext context, ActionExecuti
                     context.Result = new SnapshotsAreReadOnlyResult();
                     return Task.CompletedTask;
                 }
-                
-                _snapshotContextProvider.SetSnapshotContext(
-                    new SnapshotContext(values.FirstOrDefault()));
+
+                var snapshotContext = Enum.Parse<SnapshotUsage>(values.FirstOrDefault());
```

Comment:
We should be parsing for `true`/`false` here... and converting `true` to `SnapshotUsage.On` and anything else to `SnapshotUsage.Off`.